backend/LeetWars.Core/LeetWars.Core.DAL/Migrations/20230926185939_AddFriendships.cs

```diff
@@ -0,0 +1,1199 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace LeetWars.Core.DAL.Migrations
+{
+    public partial class AddFriendships : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DeleteData(
+                table: "UserFriendships",
+                keyColumns: new[] { "FriendshipId", "UserId" },
+                keyValues: new object[] { 1L, 11L });
```

Comment:
did we have UserFriendships before?

Reply:
this pr was partially merged before the demo, and i guess this migration could be there somehow(

Reply:
try to rollback your migration and create a new one
if not help, lets leave it as it is